CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 * Added package `s3fs` to Python environment as it is required to open Zarr datasets 
   from S3-compatible object store. #940
+* Fixed issue of harmonization of info field names of metadata (#949)
```

Comment:
Please name affected datasets. Use pattern: AEROSOL.*blabla
If this our fault? Or OpenSrearch API? If latter, inform Alison  & Richard .